### PR TITLE
Allow local overriding of grunt options (especially generation of sourcemaps)

### PR DESCRIPTION
**Pull request by rarila**
:warning:  Generation of sourcemaps is now switched off by default.

Add ``app/src/grunt.json`` and enter

```json
{
    "sourceMap": {
        "css": true,
        "js": true
    }
}
```
there to get them generated. That file is ignored by git.

:warning: Generated css and js files should be committed without sourcemaps from now on!

### CODE REVIEW

File: app/src/README.md [hunk 1]
```diff
@@ -1,17 +1,29 @@
-Node/Grunt modules for backend’s frontend workflow.
+# Bolts backend’s frontend workflow using grunt
```
Comment (GawainLynch) [inline app/src/README.md]: Pluralisation and possession is out slightly. Probably better with "Bolt's backend/frontend"

Comment (rarila) [inline app/src/README.md]: Oh, I wasn't in the mood of writing documentation yesterday. I was hoping someone with a mother tongue would be … ;-)

Comment (rarila) [inline app/src/README.md]: Ah, btw, this readme is only there out of one reason: npm complains if it is missing :-)

Comment (GawainLynch) [inline app/src/README.md]: Yeah, totally understand. Please see my comment as doing just that not nitpicking. I really hate having to correct non-native speakers English.

A native speaker on the other hand… them I nitpick :grin:

**Comment by bobdenotter** [comment]
Is this good to go, or does it need some fine-tuning?

**Comment by GawainLynch** [comment]
I have no problems with it myself, was just trying to be helpful with linguistics.

**Comment by rarila** [comment]
@bobdenotter green lights, nobody reads readmes ;-)

**Comment by bobdenotter** [comment]
Merging in, then! :train: :train:  :train: :train:  :train: :train:  :train: :train:

**Pull request merged**